src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <main class="main">
+      <label for="input-big" class="main__label main__label--big">
+        <img
+          src="images/Search.svg"
+          alt="search"
+          class="img img--big">
```

Comment:
Checklist, you need to use image as a background property
<img width="495" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/b939e5f3-ee23-4b12-b0df-9927f10a51d5">